docs/api-reference/algo-details-sdca-refs.md

```diff
@@ -0,0 +1,8 @@
+For more information, see:
+* [Scaling Up Stochastic Dual Coordinate
+  Ascent.](https://www.microsoft.com/en-us/research/wp-content/uploads/2016/06/main-3.pdf)
+* [Stochastic Dual Coordinate Ascent Methods for Regularized Loss
+  Minimization.](http://www.jmlr.org/papers/volume14/shalev-shwartz13a/shalev-shwartz13a.pdf)
+
+
+Check the See Also section for links to examples of the usage.
```

Comment:
you don't need this file. you can just move it to the end of algo-details.sdca.md. It's ok that regularization details come after this, b/c it will be a separate section.

Reply:
It looks super strange... Regularization is the reason why SDCA can exist. As you may have known, SDCA solves "Dual form" of the original optimization problem. Without regularization, that dual form may not exist. Consequently, user should learn regularization before jumping into details of SDCA.

---
In reply to: [284914930](https://github.com/dotnet/machinelearning/pull/3586#discussion_r284914930) [](ancestors = 284914930)